agile-development/collaboration/add-pairing-field-ado-cards.md

```diff
@@ -0,0 +1,89 @@
+# How to add a pairing custom field in ADO User Stories
+
+This document outlines the benefits of adding a custom field of type _Identity_ in [Azure DevOps](https://docs.microsoft.com/en-us/azure/devops/user-guide/what-is-azure-devops) (ADO) user stories, prerequisites, and a step-by-step guide.
+
+## Benefits of adding a custom field
+
+Having the names of both individuals [pairing on a story](./readme.md) visible on the ADO cards can be helpful during sprint ceremonies and lead to greater accountability by the pairing assignee. For example, it is easier to keep track of the individuals assigned stories as part of a pair during sprint planning by using the "pairing names field". During stand-up it can also help the process lead filter stories assigned to the individual (both as an owner or as a pairing assignee) and show these on the board. Furthermore, the pairing field can provide an additional data point for reports and burndown rates.
+
+## Prerequisites
+
+Prior to customizing ADO, review [Configure and customize Azure Boards](https://docs.microsoft.com/en-us/azure/devops/boards/configure-customize).
+
+In order to add a custom field to user stories in ADO changes must be made on an **Organizational setting**. This document therefore assumes use of an existing Organization in ADO and that the user account used to make these changes is a member of the [Project Collection Administrators Group](https://docs.microsoft.com/en-us/azure/devops/organizations/security/set-project-collection-level-permissions).
+
+## Step-by-step to change the organization settings
+
+### Duplicate the process currently in use
```

Comment:
I would suggest making this an ordered list instead of sub headings

Reply:
Good idea!